docs/source/cli/graph-construction/single-machine-gconstruct.rst

```diff
@@ -189,6 +189,15 @@ GraphStorm provides a set of transformation operations for different types of fe
                   "bucket_cnt": 2,
                   "slide_window_size": 10},
 
+* **Numerical Feature Transformation with Tabular Foundation Model ** will do feature engineering with `Mitra Model <https://www.amazon.science/blog/mitra-mixed-synthetic-priors-for-enhancing-tabular-foundation-models>`_ 
+The tabular foundation model generates embeddings for all columns within the input table. The Mitra model expects a label column per row, defaulting to the selection node/edge label in the graph. The ``name`` field is ``tabular``, and optional ``target_col`` specifies the label column for tabular foundation model. 
```

Comment:
The description is confusing. What will happen if users set this transformation?
Because this field is associated with one column, should values in this column be a list in each row?